### ISSUE TITLE
How to plug into yesql?

### ISSUE THREAD

**Issue by jjfine** [bugsbio/squirrel#1]
In this [yesql issue](https://github.com/krisajenkins/yesql/issues/8), you mention that squirrel easily plugs into yesql, but I can't figure out an easy way to do it. All I'm coming up with is something like this:

``` Clojure
(defquery "get-all-things-query.sql" {:connection my-connection})
(defn get-all-things []
  (map to-clj (get-all-things-query)))
```

...which would have to be done for every query. Seems like too much repetition. Any pointers?

Thanks.

**Comment by rsslldnphy** [comment]
Yeah, maybe "nicely" is a slight overstatement, but I think it's about the best we can do without changes in yesql. One thing you could do here, however, is use the `:row-fn` option, so your code could be:

``` clojure
(defquery get-all-things "get-all-things-query.sql" {:connection my-connection :row-fn to-clj})
```
